hooks/bugsnag/bugsnag.go

```diff
@@ -0,0 +1,53 @@
+package logrus_bugsnag
```

Comment:
I mean, I could remap `bugsnag.Configure()` and `bugsnag.Configuration` into this namespace, but that feels pretty ugly.